### PR TITLE
Adding default icon for the operator

### PR DESCRIPTION
**Pull request by varad-ahirwadkar**
Adding a default (dummy) icon for the operator as icon is required for submission to the community-operator repository.

### CODE REVIEW

File: bundle/manifests/rsct-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -47,8 +47,8 @@ spec:
...
   displayName: RSCT Operator for IBM Power Virtual Server
   icon:
-  - base64data: ""
-    mediatype: ""
+  - base64data: PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNTguNTEgMjU4LjUxIj48ZGVmcz48c3R5bGU+LmNscy0xe2ZpbGw6I2QxZDFkMTt9LmNscy0ye2ZpbGw6IzhkOGQ4Zjt9PC9zdHlsZT48L2RlZnM+PHRpdGxlPkFzc2V0IDQ8L3RpdGxlPjxnIGlkPSJMYXllcl8yIiBkYXRhLW5hbWU9IkxheWVyIDIiPjxnIGlkPSJMYXllcl8xLTIiIGRhdGEtbmFtZT0iTGF5ZXIgMSI+PHBhdGggY2xhc3M9ImNscy0xIiBkPSJNMTI5LjI1LDIwQTEwOS4xLDEwOS4xLDAsMCwxLDIwNi40LDIwNi40LDEwOS4xLDEwOS4xLDAsMSwxLDUyLjExLDUyLjExLDEwOC40NSwxMDguNDUsMCwwLDEsMTI5LjI1LDIwbTAtMjBoMEM1OC4xNiwwLDAsNTguMTYsMCwxMjkuMjVIMGMwLDcxLjA5LDU4LjE2LDEyOS4yNiwxMjkuMjUsMTI5LjI2aDBjNzEuMDksMCwxMjkuMjYtNTguMTcsMTI5LjI2LTEyOS4yNmgwQzI1OC41MSw1OC4xNiwyMDAuMzQsMCwxMjkuMjUsMFoiLz48cGF0aCBjbGFzcz0iY2xzLTIiIGQ9Ik0xNzcuNTQsMTAzLjQxSDE0MS42NkwxNTQuOSw2NS43NmMxLjI1LTQuNC0yLjMzLTguNzYtNy4yMS04Ljc2SDEwMi45M2E3LjMyLDcuMzIsMCwwLDAtNy40LDZsLTEwLDY5LjYxYy0uNTksNC4xNywyLjg5LDcuODksNy40LDcuODloMzYuOUwxMTUuNTUsMTk3Yy0xLjEyLDQuNDEsMi40OCw4LjU1LDcuMjQsOC41NWE3LjU4LDcuNTgsMCwwLDAsNi40Ny0zLjQ4TDE4NCwxMTMuODVDMTg2Ljg2LDEwOS4yNCwxODMuMjksMTAzLjQxLDE3Ny41NCwxMDMuNDFaIi8+PC9nPjwvZz48L3N2Zz4=
```
Comment (mkumatag) [inline bundle/manifests/rsct-operator.clusterserviceversion.yaml:50]: from where you picked up this icon? what is the usage licence for this image?

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: bundle/manifests/rsct-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (varad-ahirwadkar) [reply bundle/manifests/rsct-operator.clusterserviceversion.yaml:50]: I have picked the icon from [#operator-icon](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#operator-icon). And this is the default icon.

**Review by mkumatag — APPROVED** [review]
/lgtm

**Pull request merged**